src/main/java/com/denizenscript/depenizen/bukkit/events/shopkeepers/ShopKeeperTradeCompletedScriptEvent.java

```diff
@@ -46,14 +44,14 @@ public ScriptEntryData getScriptEntryData() {
     @Override
     public ObjectTag getContext(String name) {
         return switch (name) {
-            case "recipe" -> ShopkeepersBridge.tradingRecipeToList(event.getTradingRecipe());
+            case "recipe" -> ShopkeepersBridge.tradingRecipeToList(event.getCompletedTrade());
```

Comment:
Diiid you verify this compiles & works? might be missing something, but it looks like that method returns a `ShopkeeperTradeEvent` (which you can get a recipe from), not the recipe itself - see the [source](https://github.com/Shopkeepers/Shopkeepers/blob/d2106f0f81f73208c8d947b76e44d5408696f921/modules/api/src/main/java/com/nisovin/shopkeepers/api/events/ShopkeeperTradeCompletedEvent.java#L39-L41).